RX interruption catched
TASK_ALLOCATION_SEND
Got slot: 0 
Task total size (txt+bss): 3124 
Task addr: 2147522956 
Task slot: 0 
Task slot TaskAddr: 2147522956 
RX interruption catched
TASK_ALLOCATION_FINISHED
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um TASK_START!
RX interruption catched
Chegou um TASK_RUN!
Starting Task 4  from app 1 
taskSlot run: 0 
SendingQueue_front: 1 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
Task P4 INIT
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 3 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 5 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 1 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 2 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 3 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 2000350 
TotalEnergy: 21059648 
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 998688 
TotalEnergy: 45062274 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 999799 
TotalEnergy: 45144731 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1040021 
TotalEnergy: 47194785 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 959904 
TotalEnergy: 43800327 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 999799 
TotalEnergy: 45372716 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 999972 
TotalEnergy: 45378606 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000001 
TotalEnergy: 45382497 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1002425 
TotalEnergy: 45523467 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 130896 
TotalEnergy: 10782266 
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 704980 
TotalEnergy: 33730262 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 979926 
TotalEnergy: 44585550 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 997779 
TotalEnergy: 45302852 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000001 
TotalEnergy: 45381790 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000025 
TotalEnergy: 45381231 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1001213 
TotalEnergy: 45441152 
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1050727 
TotalEnergy: 47610509 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 950006 
TotalEnergy: 43409596 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 997779 
TotalEnergy: 45295754 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000001 
TotalEnergy: 44917808 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000001 
TotalEnergy: 44678613 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000001 
TotalEnergy: 44678598 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 999900 
TotalEnergy: 44674698 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000126 
TotalEnergy: 44683972 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 999958 
TotalEnergy: 44676639 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 999900 
TotalEnergy: 44674723 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 5156250 
total instructions~~~> 1000001 
TotalEnergy: 44678598 
TX interruption catched
API_Try2Send failed - empty SendQueue!
